tests/plugins/test_bump_release.py

```diff
@@ -424,3 +432,113 @@ def CGInitBuild(self, cg_name, nvr_data):
         if reserve_build and reactor_config_map and not next_release['scratch']:
             assert plugin.workflow.reserved_build_id == build_id
             assert plugin.workflow.reserved_token == token
+
+    @pytest.mark.parametrize('reserve_build, init_fails', [
+        (True, RuntimeError),
+        (True, koji.GenericError),
+        (True, None),
+        (False, None)
+    ])
+    @pytest.mark.parametrize('next_release', [
+        {'builds': [], 'expected': '1.1', 'scratch': False},
+        {'builds': ['1.1', '1.2'], 'expected': '1.3', 'scratch': False},
+        {'builds': [], 'expected': '1.scratch', 'scratch': True},
+        {'builds': ['1.1', '1.2'], 'expected': '1.scratch', 'scratch': True},
```

Comment:
Minor: these are slightly easier to follow if 'expected' is listed last, i.e. given these circumstances, the expectation is: ...

Reply:
ok will put expected last